src/scss/2022/components/_pie-chart.scss

```diff
@@ -0,0 +1,276 @@
+    border: 1px solid var(--color-white);
+  
+    &::after {
+      content: '';
+      display: block;
+      padding-top: 100%;
+    }
+  }
+
+  &__divider {
+    position: absolute;
+    top: 0;
+    left: calc(50% - 0.5px);
+    width: 1px;
+    height: 50%;
+    background-color: white;
+    z-index: 200;
+    transform-origin: 50% 100%;
+    
+    &--object-1 {
+      transform: rotate(var(--turn-1));
+    }
+    &--object-2 {
+      transform: rotate(var(--turn-2));
+    }
+    &--object-3 {
+      transform: rotate(var(--turn-3));
+    }
+  }
+
+  &__piece {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    &--object-1 {
+      background-color: var(--color-black);
+      mask-image: conic-gradient(
+        #000 0deg,
+        #000 var(--turn-1),
+        transparent var(--turn-1)
+      );
+    }
+    
+    &--object-2 {
+      $size: 6;
+      background: fills.diagonal(color.alpha('gray', 0.8), 6, 0.5) fixed;
+      mask-image: conic-gradient(
+        transparent 0deg,
+        transparent var(--turn-1),
+        #000 var(--turn-1),
+        #000 var(--turn-2),
+        transparent var(--turn-2)
+      );
+    }
+    
+    &--object-3 {
+      background: fills.dots(color.alpha('gray', 0.8), 12, 1) center / 6px 6px fixed;
+      mask-image: conic-gradient(
+        transparent 0deg,
+        transparent var(--turn-2),
+        #000 var(--turn-2),
+        #000 var(--turn-3),
+        transparent var(--turn-3)
+      );
+    }
+  }
+  
+  &__keys {
+    margin-top: var(--space-lg);
+    border: 1px solid var(--color-gray);
+    transform: translateX(0.5rem);
+    
+    @media (min-width: mq(sm)) {
+      border: none;
+      margin: auto;
+      transform: unset;
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      z-index: 300;
+    }
+  }
+  
+  &__key {
+    @include typography.style("body-default");
+    display: flex;
+  
+    & + & {
+      border-top: 1px solid var(--color-gray);
+    }
+  
+    @media (min-width: mq(sm)) {
+      display: inline-block;
+      background-color: var(--color-black);
+      border: 1px solid var(--color-gray);
+      position: absolute;
+      padding: var(--space-xs) var(--space-sm);
+
+      &::before,
+      &::after {
+        content: '';
+        background-color: var(--color-gray);
+        position: absolute;
+        
+      }
+      
+      &::before {
+        height: 2rem;
+        width: 1px;
+        top: 100%;
+        left: 50%;
+      }
+
+      &::after {
+        height: 4px;
+        width: 4px;
+        border-radius: 50%;
+        top: calc(100% + 2rem);
+        left: calc(50% - 2px);
+      }
+    }
+    
+    @media (min-width: mq(md)) {
+      padding: var(--space-sm) var(--space-md);
+    }
+
+    @media (min-width: mq(lg)) {
+      white-space: nowrap;
+      
+      &::before {
+        height: 1px;
+        width: 2rem;
+        left: 100%;
+        top: calc(50% - 0.5px);
```

Comment:
I think you can also do this with 
```
	top: 50%;
	transform: translateY(-50%);
```

Same result, maybe less headmath for each object's height. Downside is it adds a line of code, but might be easier to maintain.

Reply:
Regarding maintenance, once these sites go live we pretty much never need to maintain them. They’re one and done kinds of sites, even though they all live in this large repo.

And the `top` + `transform` makes sense… thoughI have reservations about that technique, but I will have less once the transform values get their own properties.